codebuild/release/upload_artifacts.yml

```diff
@@ -37,4 +37,4 @@ phases:
         mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.2:get \
           -DrepoUrl=https://aws.oss.sonatype.org \
           -Dartifact=com.amazonaws:aws-encryption-sdk-java:${VERSION}:jar:javadoc
-      - gh release upload v${VERSION} ~/.m2/repository/com/amazonaws/aws-encryption-sdk-java/${VERSION}/*.jar
+      - gh release create v${VERSION} ~/.m2/repository/com/amazonaws/aws-encryption-sdk-java/${VERSION}/*.jar -d -F CHANGELOG.md -t "AWS Encryption SDK ${VERSION} Release -- ${date '%y-%m-%d'}"
```

Comment:
Have you tested this in our CodeBuild image? My Mac doesn't like this `date` invocation.

Reply:
I tested it on an Ubuntu instance but not our specific codebuild image

Reply:
This works on macOs and on Ubuntu $(date +%Y-%m-%d). Updating